### ISSUE TITLE
There should be a way to exit a game / switch games

### ISSUE THREAD

**Issue by hoverbird** [ckolderup/picaro#26]
When run in full-screen mode, there's no URL bar to use as a janky game-picker.  We'll want some way to make that happen.

**Comment by hoverbird** [comment]
This might be the biggest blocker for 1.0.  When the app is in full-screen mode, not being able to show off a variety of games really limits our ability to demo Picaro as a framework.

**Comment by ardubs** [comment]
cosign

how do we feel about the idea of games loading kind of passively -- that is, in the same play window as the previous game? you could even make it so that games have transitional text, like an "in_greeting" and "out_farewell" field. could be kind of neat for games to "talk" to each other that way, but it could also be too specific a notion for a general spec.

in any case, i defs think the player experience from game to game is worth further rumination. thoughts?

**Comment by ckolderup** [comment]
I don't see why we don't just use the navigation built into the Sinatra app with a back button and possibly a link to the "recent" games (and maybe eventually a "popular" games page)?

**Comment by JoeBernardi** [comment]
I feel like loading games in the same play window could get kind of confusing. At the very least, I think we'd need a delineation in the UI that's way stronger than any of the ones we've got now.

Wondering what David's thoughts are on this, but I think I'd like to keep non-game elements out of the play screen as much as possible. Maybe we could fit a link back to an archive/recent/popular(/recommended?) page in the game UI somewhere, but I'm generally okay with it being a totally separate page.

I definitely agree that we'll want the endgame UX centered around keeping the player around for another game, but another part of me wants the game creator to have as much agency over their endgame experience as possible. It's a tough problem.

**Comment by ckolderup** [comment]
Well, if we're sticking with some predefined italicized text about "Please share this story on Facebook or Twitter" or whatever, it'd be easy to append that with "or go back to find a new game". But that only solves the problem of going back on completion— we need some way to trigger history(-1) at any point...

**Comment by hoverbird** [comment]
Yeah, some way to leave the game and get back to the interface for finding new games (without finishing the game I'm in) is definitely a requirement.

The ability to quickly switch between games I've played in the past without leaving the play window would be really great, though that's not as pressing for me.

**Comment by JoeBernardi** [comment]
I totally agree that that stuff has got to be in there somehow (doubly the "recent games" thing, if we end up implementing that savegame idea that Patrick floated awhile back), though I'm struggling to figure out how best to do it without it getting in the way of the game interface. Maybe an extra action-looking button that pops up a list of recent games and a link to the archive page? That would clutter the action bar on an iphone, though. Hm.

**Comment by irondavy** [comment]
What do you guys think of tugging down on the room bar revealing something? Like the notification tab in iOS. I know it's not super discoverable but we might be able to make it make sense when a game is first loaded, like the whole thing slides up? I don't know how easy this would be to implement, technically.

**Comment by ckolderup** [comment]
That might work, but is having tugging down on the room bar vs tugging down on the main area (to scroll back through the log) confusing?

**Comment by irondavy** [comment]
Yeah, that's a concern I had as well.

We could make it that the first message printed is something like "This is Leaf by Rob Dubbin. [Return to the game list]." Like the equivalent of Casey's suggestion for the end.
